component/prometheus/scrape/scrape.go

```diff
@@ -119,7 +119,7 @@ var (
 
 // New creates a new prometheus.scrape component.
 func New(o component.Options, args Arguments) (*Component, error) {
-	flowAppendable := prometheus.NewFanout(args.ForwardTo, o.ID)
+	flowAppendable := prometheus.NewFanout(args.ForwardTo, o.ID, o.Registerer)
```

Comment:
This registerer will already be wrapped with a label containing the component ID, right?

Reply:
Yes